packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Indemnite.js

```diff
@@ -20,9 +20,10 @@ function StepIndemnite({ form }) {
   const { indemniteLegale, formuleLegale } = getIndemniteFromFinalForm(form);
   return (
     <>
-      <IndemniteLegale indemnite={indemniteLegale} formula={formuleLegale} />
-      <br />
-      <br />
+      <IndemniteLegale
+        indemniteLegale={indemniteLegale}
```

Comment:
not sure we need to add `Legale` suffix to props since it's the component that display <IndemnitéLegale />`

Reply:
Yep I hesitated for this one but I did it in the end because this way it is consistent in all the calculator and clearly distinguished from the Conventionnelle one. But yeah I could keep it `indemnite` in this file if you prefer

Reply:
d'acc, on laisse indemniteLegale